docs/files.md

```diff
@@ -263,6 +298,57 @@ For more information, see the [API documentation](https://docs.box.com/reference
 client.files.getEmbedLink('12345', callback);
 ```
 
+Lock a File
+-----------
+
+A file can be locked, which prevents other users from editing the file, by calling the
+[`files.lock(fileID, expiresAt, isDownloadPrevented, callback)`](http://opensource.box.com/box-node-sdk/Files.html#lock)
```

Comment:
Looks like you put expiresAt and isDownloadPrevented into an options object. The example below looks OK, but the signature above is wrong.